dataRetriever/resolvers/baseSimpleResolver.go

```diff
@@ -0,0 +1,17 @@
+package resolvers
+
+import (
+	"github.com/ElrondNetwork/elrond-go/storage"
+)
+
+type baseSimpleResolver struct {
+	storer storage.Storer
+}
+
+func (bsr *baseSimpleResolver) getFromStorage(key []byte, _ uint32) ([]byte, error) {
+	return bsr.storer.SearchFirst(key)
```

Comment:
You can call bsr.searchFirst(key), as it does the same thing, and this way will avoid to have two identical methods

Reply:
I need those 2 identical methods as to satisfy the newly introduced interface that has a different implementation on `baseFullHistoryResolver` struct. For simplicity, I would keep this code as it is.

Reply:
ok